perfkitbenchmarker/linux_packages/silo.py

```diff
@@ -33,6 +33,9 @@ def _Install(vm):
   vm.RemoteCommand('git clone {0} {1}'.format(GIT_REPO, SILO_DIR))
   vm.RemoteCommand('cd {0} && git checkout {1}'.format(SILO_DIR,
                                                        GIT_TAG))
+  # This is due to a failing clone command when executing behind a proxy.
+  # Replacing the protocol to https instead of git fixes the issue.
+  vm.RemoteCommand('git config --global url."https://".insteadOf git://')
```

Comment:
Is the problem here the use of port 8080?  IT seems like other places curl is used to to proxy via http but here you are trying to move to ssh.

LGTM but just wanted to make sure I understood.